src/app/sign-in/pages/sign-in/sign-in.component.html

```diff
@@ -42,6 +41,7 @@
                   [disabled]="loading"
                   *ngIf="!show2FA"
                   i18n="@@ngOrcid.signin.button"
+                  id="signing-button"
```

Comment:
I think it should be `signin-button` instead

Reply:
Fixing on #618